### PR TITLE
Add 2 new scripts to check subset coverage

### PR DESCRIPTION
**Pull request by nyshadhr9**

### CODE REVIEW

**Comment by davelab6** [comment]
cc @felipesanches @m4rc1e @graphicore

File: tools/font_weights_coverage.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+"""Tool to check codepoint coverage in all font weights.
+
+Ex: If FamilyName-Regular.ttf supports codepoints A-D
+       FamilyName-Bold.ttf supports codepoints B-E
+       FamilyName-Light.ttf supports codepoints A-E
+
+$ python tools/font_wieghts+coverage.py ofl/familyname
```
Comment (felipesanches) [inline tools/font_weights_coverage.py]: there's a typo here

Comment (felipesanches) [inline tools/font_weights_coverage.py]: actually two typos :-)

**Comment by davelab6** [comment]
Thanks, fixed in d221b49

**Pull request merged**